modules/op-enabling-tekton-chains-to-operate-only-in-selected-namespaces.adoc

```diff
@@ -0,0 +1,39 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-tekton-chains-to-operate-only-in-selected-namespaces_{context}"]
+= Enabling {tekton-chains} to operate only in selected namespaces
+
+By default, the {tekton-chains} controller monitors resources in all namespaces. You can customize {tekton-chains} to run only in specific namespaces, which provides granular control over its operation.
+
+.Prerequisites
+* You are logged in to your {OCP} cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. In the `TektonConfig` CR, in the `chain` section, add the `--namespace=` argument to contain the namespaces that the controller should monitor.
```

Comment:
Switch to unordered since this is a single-step procedure:
```suggestion
* In the `TektonConfig` CR, in the `chain` section, add the `--namespace=` argument to contain the namespaces that the controller should monitor.
```